example/server/config/config.go

```diff
@@ -0,0 +1,45 @@
+package config
+
+import (
+	"os"
+	"strings"
+)
+
+const (
+	// default port for the http server to run
+	DefaultIssuerPort = "9998"
+)
+
+type Config struct {
+	Port         string
+	RedirectURIs []string
+	UsersFile    string
+	Issuer       string
+}
+
+// FromEnvVars loads configuration parameters from environment variables.
+// If there is no such variable defined, then use default values.
+func FromEnvVars(defaults *Config) *Config {
+	if defaults == nil {
+		defaults = &Config{}
+	}
+	cfg := &Config{
+		Port:         defaults.Port,
+		RedirectURIs: defaults.RedirectURIs,
+		UsersFile:    defaults.UsersFile,
+		Issuer:       defaults.Issuer,
+	}
+	if value, ok := os.LookupEnv("PORT"); ok {
+		cfg.Port = value
+	}
+	if value, ok := os.LookupEnv("USERS_FILE"); ok {
+		cfg.UsersFile = value
+	}
+	if value, ok := os.LookupEnv("REDIRECT_URIS"); ok {
```

Comment:
Please keep the environment variable name `REDIRECT_URI`, so we don't break existing uses of this variables.
```suggestion
	if value, ok := os.LookupEnv("REDIRECT_URI"); ok {
```